jarviscli/plugins/joke_of_day.py

```diff
@@ -0,0 +1,52 @@
+from plugin import plugin, require
+import requests
+from colorama import Fore
+from plugins.animations import SpinnerThread
+
+
+@require(network=True)
+@plugin('jokeofday')
+class joke_of_day:
+    """
+    Provides you with a joke of day to help you laugh amidst the
+    daily boring schedule
+
+    Enter 'jokeofday' to use
+
+    """
+
+    def __call__(self, jarvis, s):
+        jarvis.say("Welcome To The Plugin Joke Of Day!", Fore.CYAN)
+        print()
+        joke_fetch = self.get_joke(jarvis)
+        if joke_fetch is not None:
+            self.joke(jarvis, joke_fetch)
+
+    def get_joke(self, jarvis):
+        spinner = SpinnerThread('Fetching ', 0.15)
+        while True:
+            joke = jarvis.input(
```

Comment:
Don't actually see the use for this user input. If user enters 'jokeofday' - he probably want's the joke of the day :).

Or do you e.g. plan to add features like 'random joke'? If yes, you could also add two plugins: "joke" for e.g. random joke and 'joke day" for the joke of the day.